### ISSUE TITLE
unknown config file keywords are silently ignored

### ISSUE THREAD

**Issue by jmichel80** [michellab/Sire#41]
This makes it easy to make setup errors by for instance writing a keyword with a typo.  

It may be more sensible as part of our plans to implement sanity checks to at least raise a warning if the user has passed a keyword that is unrecognised by one app.

**Comment by ppxasjsm** [comment]
Yes definitely, and it didn't help that I renamed minimize -> minimise.
